custom/opm/filters.py

```diff
@@ -51,6 +70,56 @@ def get_hierarchy():
     return hierarchy
 
 
+def user_data_as_hierarchy():
```

Comment:
Since CMR and BPR are split by individual pregnant cases, no direct comparison with HSR is necessary/possible, unlike HSR and AWW-report. Currently there is no consistency between HSR and AWWR. AWWR correctly splits same AWC-names into seperate columns based on the owner-id, both in filtering and in the resulting report rows. 
This PR lets HSR split same names into separate rows consistently as AWWR. The filters still show one AWC though. You can see all of this by using this filters in AWWR and HSR (with this PR, without this PR) - Wazirganj > Nawada > Manjhauli Harijan Tola.
So in a way this PR reduces inconsistencies. We can fix the filters too, but I thought this is not super-essential, since there are only two pairs with same names in around 2000 AWCs